app/models/service_template_ansible_playbook.rb

```diff
@@ -51,16 +51,22 @@ def self.prepare_job_template_and_dialog(action, service_name, description, conf
   private_class_method :prepare_job_template_and_dialog
 
   def self.create_job_template(name, description, info)
-    playbook = ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook.find(info[:playbook_id])
-    # tower = playbook.manager
+    playbook = ManageIQ::Providers::AnsibleTower::AutomationManager::Playbook.find(info[:playbook_id])
+    tower = playbook.configuration_script_source.manager
 
-    # params = {
-    #   :name         => name,
-    #   :description  => description || '',
-    #   :extra_vars   => info[:variables] || {},
-    #   :inventory_id => playbook.inventory_root_group,
-    # }
-    # tower.class.create_in_provider(tower, params)
+    params = {
+      :name               => name,
+      :description        => description || '',
+      :extra_vars         => info[:variables] || {},
+      :project            => playbook.configuration_script_source.manager_ref,
+      :playbook           => playbook.manager_ref,
+      :inventory          => playbook.inventory_root_group.ems_ref,
+      :credential         => info[:credential] || '',
+      :cloud_credential   => info[:cloud_credential] || '',
+      :network_credential => info[:network_credential] || ''
+    }
```

Comment:
@bdunne I think these `ask_` flags should be hard coded to true. This is an internal job template and we should not need to ask user to decide the flag through UI.